src/js/components/DataChart/XAxis.js

```diff
@@ -1,32 +1,30 @@
 import React, { forwardRef } from 'react';
 import { Box } from '../Box';
 
-const XAxis = forwardRef(
-  ({ chartProps, data, renderValue, serie }, ref) => {
-    // pull the x-axis values from the first chart, all should have the same
-    const [axisValues] = (Array.isArray(chartProps[0])
-      ? chartProps[0][0]
-      : chartProps[0]
-    ).axis;
+const XAxis = forwardRef(({ chartProps, data, renderValue, serie }, ref) => {
```

Comment:
Why did this file change?

Reply:
I believe it is a formatting change implemented by the linter on commit. I did not touch the file.

Reply:
It looks like these changes came in on this commit which was a merge of master: https://github.com/grommet/grommet/pull/4426/commits/8873ff0c64f3ab9836effa8b71540f6762d6b114

Reply:
Can you merge master properly so we won't see those changes? If it was already done on a master's commit it shouldn't appear on this PR.

Reply:
Will give it another go.